config/find-eu-exit-guidance-business-email-signup.yml

```diff
@@ -16,7 +16,7 @@ details:
     radio_button_name: "yes"
     topic_name: "appear in find eu exit guidance business finder"
     prechecked: true
-  subscription_list_title_prefix: 'Find EU Exit guidance for your business'
+  subscription_list_title_prefix: 'Find EU Exit guidance for your organisation'
```

Comment:
We don't want to change this - it will mean email subscriptions will no longer work as the slug of the email is created from `subscription_list_title_prefix`